src/app/create/codebases/services/github.service.ts

```diff
@@ -67,9 +65,7 @@ export class GitHubService {
     if (this.cache.has(url)) {
       return this.cache.get(url);
     } else {
-      let res = this.getHeaders(this.getUserName(fullName))
```

Comment:
This code is very dangerous like this, as you have no guarantee that the token is fetched before you start trying to use it. You need to make sure that you have the token before you start trying to use it, which is what the previous code was doing.